.github/workflows/release.yml

```
@@ -22,4 +22,6 @@ jobs:
     - name: Publish
       env:
         NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
-      run: npm publish --access public --@wmde:registry=https://registry.npmjs.org/
+      run:
```

```suggestion
      run: |
```

Needs a pipe here to allow multi line

---

I had one job to copy two lines, and failed

---

> I had one job to copy two lines

And that's where you got it wrong. You have 3 lines to copy. :)